dockerize.sh

```diff
@@ -3,17 +3,29 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 IMAGE_NAME="boresha/agritrader"
 DOCKER_HOME="/agritrader"
-ARG_NUM=2
 
 if [ "$1" == "init" ]; then
-	docker build -t $IMAGE_NAME $DIR
+	docker build -t "$IMAGE_NAME" "$DIR"
 elif [ "$1" == "install" ]; then
-	docker run --rm -v $DIR:$DOCKER_HOME -t \
-		$IMAGE_NAME yarn install
+	docker run --rm \
+		-v "$DIR":"$DOCKER_HOME" \
+		"$IMAGE_NAME" yarn install
+elif [ "$1" == "adb" ]; then
```

Comment:
do you think it would be misleading that using the adb command in the dockerize script only runs `adb devices`? Would there be other adb commands that would be useful? If yes, might be a good idea to make it similar to the run command that uses the other arguments?